### ISSUE TITLE
archive/delete?

### ISSUE THREAD

**Issue by amchagas** [ohwmakers/feeling-responsive#1]
should we archive or delete this one? as it hasn't been properly deployed/used?

**Comment by vektorious** [comment]
Yep. :+1:, let's delete it as soon as the new website is online.
